weave

```diff
@@ -801,6 +857,7 @@ case "$COMMAND" in
     expose)
         collect_cidr_args "$@"
         shift $CIDR_COUNT
+        [ $CIDR_COUNT -gt 0 ] || { get_ipam_cidr weave && expose $CIDR_ARGS ; }
```

Comment:
incidentally, this code is just wrong. As it stands it ends up associating the IP with the weave container. But the address is in fact attached to the bridge, which survives weave restarts.